docs/lib/storage/fragments/android/list.md

```diff
@@ -2,20 +2,13 @@ You can list all of the objects uploaded.
 ```java
     private void listFiles() {
         Amplify.Storage.list(
-                "",
-                new ResultListener<StorageListResult>() {
-                    @Override
-                    public void onResult(StorageListResult storageListResult) {
-                        for(StorageListResult.Item item : storageListResult.getItems()) {
-                            Log.i("StorageQuickStart", "Item: " + item.getKey());
-                        }
-                    }
-
-                    @Override
-                    public void onError(Throwable error) {
-                        Log.e("StorageQuickStart", error.getMessage());
-                    }
+            "",
```

Comment:
Hm, whats the `""` supposed to be again? Should this be a non-empty string? If not, maybe a `"", /* quick explanation of why this is blank */` for the doc